model/BUILD

```diff
@@ -4,6 +4,7 @@ go_library(
     name = "go_default_library",
     srcs = [
         "controller.go",
+        "ingress.go",
         "registry.go",
```

Comment:
So, we need something in the model (`RoutingRule`?) that the envoy ingress watcher can consume, and the kube ingresses converted to. Is there already any work in that direction?